examples/ecs/kuma-cp-remote.yaml

```diff
@@ -280,6 +345,9 @@ Resources:
           Essential: true
           User: '5678'
           Environment:
+            - Name: KUMA_DATAPLANE_RUNTIME_TOKEN
+              # generated with "wget --header='Content-Type: application/json' --post-data='{\"mesh\": \"default\", \"type\": \"ingress\"}' -qO- http://localhost:5681/tokens"
+              Value: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJOYW1lIjoiIiwiTWVzaCI6ImRlZmF1bHQiLCJUYWdzIjp7fSwiVHlwZSI6ImluZ3Jlc3MifQ.62MCnX4pCJztiLsqDSOopKxCDbxGstFrGZ1jboPOcoA
```

Comment:
How does it work? The signing key for dataplane token is changing with every restart if you don't use persistent storage.

Should we instruct the users to replace it with the value generated after deploying global CP and generating the token?

Reply:
Oh that now makes much more sense, maybe I should split the ingress to a separate template and instruct to actually generate the token as with the regular workload.

Reply:
I think that makes sense